modules/apps/asset/asset-categories-admin-web/src/main/resources/META-INF/resources/js/ActionsComponentPropsTransformer.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ItemSelectorDialog} from 'frontend-js-web';
+
+const ACTIONS = {
+	deleteVocabularies({
+		deleteVocabulariesURL,
+		portletNamespace,
+		viewVocabulariesURL,
+	}) {
+		const vocabulariesFm = document.createElement('form');
```

Comment:
nit: would be best to use `Form` suffix instead of `Fm`.

Reply:
Done